models/business_models/product/newtab/newtab.yml

```diff
@@ -0,0 +1,5 @@
+version: 2
+
+models:
+  - name: newtab_visits
+    description: New Tab Visits
```

Comment:
"Visits on the newtab page, one row per visit per-day (identified by the visit id)."

Might as well re-use the description from bqetl :) 
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/newtab_visits_v1/metadata.yaml